src/core/distributions/math/GeometricDistribution.cpp

```diff
@@ -45,15 +45,15 @@ GeometricDistribution* GeometricDistribution::clone( void ) const
 double GeometricDistribution::computeLnProbability( void ) 
 {
     
-    double v = *value;
+    long v = *value;
     
     // check that the value is inside the boundaries
     if ( v < 0.0 )
     {
         return RbConstants::Double::neginf;
     }
     
-    return RbStatistics::Geometric::lnPdf(*value, p->getValue());
+    return RbStatistics::Geometric::lnPdf(int(v), p->getValue());
 }
```

Comment:
Should we use `long` or `long long` here also?

Reply:
Well, since `value` is of type `long*` as defined by `TypedDistribution<long>`, I would stick with long for consistency.

Reply:
OK, maybe change `int` to `long` then.

Reply:
Or replace `int(v)` with just `v`.

Reply:
OK, I changed that throughout for the geometric distribution.